src/EnergyPlus/DataGlobals.hh

```diff
@@ -54,7 +54,6 @@
 
 // EnergyPlus Headers
 #include <EnergyPlus/EnergyPlus.hh>
-#include <EnergyPlus/Data/EnergyPlusData.hh>
```

Comment:
This was causing a circular inclusion.  And it's not needed.  If, for some reason, DataGlobals needs something from the Data.hh file, it can forward declare it.